API_doc/Overview.md

```diff
@@ -23,16 +23,19 @@ Using the above functions it's possible for an application to persistently use o
 
 A configuration is represented by a series of instructions.
 The most basic instruction is a setting, which can alias another setting.
+You can also unset an alias or remove an existing setting.
 
 ```cpp
-void $LIB_NAME::Config:addSetting(char const *name, char const *value);
-void $LIB_NAME::Config:addSettingAlias(char const *name, char const *aliasName);
+void $LIB_NAME::Config::addSetting(char const *name, char const *value);
+void $LIB_NAME::Config::addSettingAlias(char const *name, char const *aliasName);
+void $LIB_NAME::Config::unsetAlias(char const *aliasName);
+void $LIB_NAME::Config::removeSetting(char const *name);
```

Comment:
I'm not to sure about this, since quoting above:
```A configuration is represented by a series of instructions.```
This is basically a way to hide a setting. Which could be interesting but I feel that maybe it would be better to have undo capabilities instead